packages/Interface/test/tstKokkosView.cpp

```diff
@@ -210,14 +210,60 @@ TEUCHOS_UNIT_TEST_TEMPLATE_2_DECL( View, basic_reduce_kernel, Scalar,
     typename ViewType::HostMirror host_data =
         Kokkos::create_mirror_view( data );
     Kokkos::deep_copy( host_data, data );
-    double test_sum = 0.0;
+    Scalar test_sum = 0.0;
     for ( int i = 0; i < size; ++i )
     {
         test_sum += host_data( i );
     }
     TEST_EQUALITY( test_sum, sum );
 }
 
+//---------------------------------------------------------------------------//
+// Test for creating a view and running a basic scan kernel
+TEUCHOS_UNIT_TEST_TEMPLATE_2_DECL( View, basic_scan_kernel, Scalar, DeviceType )
```

Comment:
What is the point of this unit test?

Reply:
this is from when I was trying to figure out if the cuda 9 issues came from parallel_scan. This doesn't belong in this PR so I am removing the test.